### PR TITLE
MIDI-144: old Quantizers

### PR DESCRIPTION
**Pull request by WojciechMat**
### MidiQuantizers
I am moving all the quantizers used in our experiments to this repo.

#### RelativeTimeQuantizer
Quantizers that depend on sequential occurence of notes. They use `dstart`, which is the distance between start of the current note and start of the previous one, as a timing measure.
#### AbsouteTimeQuantizer
Grid-based quantizers. Apperance of notes does not depend on other notes.

#### Dashboard
```sh
PYTHONPATH=. streamlit run --server.port 4010 dashboards/quantizer_review.py
```
https://0c3c-46-205-140-240.ngrok-free.app/

### CODE REVIEW

**Comment by roszcz** [comment]
Please add details about what each quntizer is doing on the dashboard :)

File: artifacts/bin_edges.yaml
Comment (roszcz) [inline artifacts/bin_edges.yaml:1]: I think you can include the code necessary to calculate values in this file, just put it in `scripts/` for easier reference :)

File: dashboards/quantizer_generator.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+
+class AbsoluteTimeQuantizerFactory(QuantizerFactory):
+    quantizer_desc = """
+    - `pitch`: uses all 88 pitch values
+    - `velocity`: quantization using bins from artifacts/bin_edges.yaml
+    - timing:
+        - `start`: quantizes start into `n_start_bins` bins, evenly distributed across `sequence_duration`.
+        If the piece is longer that `sequence_duration`, all late notes will land in the last bin.
+        - `duration`: quantization using bins from artifacts/bin_edges.yaml
```
Comment (roszcz) [inline dashboards/quantizer_generator.py:36]: Some explanation about how those values were calculate would be useful, just pointing the reader to a file full of numbers is not really providing much insight :)

**Review by roszcz — APPROVED** [review]

File: artifacts/bin_edges.yaml
Comment (WojciechMat) [reply artifacts/bin_edges.yaml:1]: Sure, let me just find it :sweat_smile:

**Pull request merged**